components/motor/gpio/basic_test.go

```diff
@@ -185,7 +196,17 @@ func TestMotorDirPWM(t *testing.T) {
 		test.That(t, mustGetGPIOPinByName(b, "1").Get(context.Background()), test.ShouldEqual, false)
 		test.That(t, mustGetGPIOPinByName(b, "3").PWM(context.Background()), test.ShouldEqual, .5)
 
+		errChan := make(chan error)
+		go func() {
+			goForErr := m.GoFor(ctx, 50, 100, nil)
```

Comment:
note that due to another bug (pr coming once i finish testing), Stop() won't actually cancel GoFor. See RSDK-2107.
I suspect that this may introduce test flakiness as the goroutine will keep running and may conflict with other tests since motor `m` is reused.

Reply:
I take that back. Stop() will trigger GoFor() to complete. The PR I mentioned will not be merged.